docs/components/attention/react.md

```diff
@@ -32,16 +32,29 @@ function Example() {
         ref={targetEl}
         onMouseEnter={() => setShow(true)}
         onMouseLeave={() => setShow(false)}
+        onMouseEnter={() => setShow(true)}
+        onMouseLeave={() => setShow(false)}
+        onFocus={() => setShow(true)}
+        onBlur={() => setShow(false)}
+        onKeyDown={handleKeyDown}
+        tabIndex={0}
       >
-        hover this for useless info
+        <button
+          aria-describedby='tooltip-bubbletext'
+          aria-expanded='true'
+          type='button'
+          className='bg-transparent'
+        >
+          hover this for useless info
+        </button>
       </h1>
```

Comment:
Do we need to nest a button inside h1? Maybe just having a `<Button>` here would suffice? Is it something you got feedback on from the a11y team?

Reply:
This is implemented according to the a11y-team's recommended solution for tooltip. :)